frontend/src/components/activity/DaySwitcher.vue

```diff
@@ -79,6 +79,11 @@ export default {
         .days()
         .items.map((day) => this.getDayObject(day))
     },
+    sortedDays() {
+      const days = [...this.days]
+      days.sort((a, b) => new Date(a.value.start) - new Date(b.value.start))
+      return days
+    },
```

Comment:
minor:
with node 20 you can maybe do now:
```suggestion
 sortedDays() {
      return days.toSorted((a, b) => new Date(a.value.start) - new Date(b.value.start))
    },
```
And why not sort by day number, the api already has this information.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted

Reply:
Would this also be polyfilled? Because less than general availability is in my opinion not enough.